pkg/view/flexlayout/flexlayout_test.go

```diff
@@ -70,7 +70,7 @@ func TestFlexLayout_add_button(t *testing.T) {
 	got := fl.ToComponent("Title")
 	expected := component.NewFlexLayout("Title")
 	buttonGroup := component.NewButtonGroup()
-	buttonGroup.AddButton(component.NewButton("button", action.Payload{}))
+	buttonGroup.AddButton(*component.NewButton("button", action.Payload{}))
```

Comment:
Same comment as other places, make AddButton accpet the return of NewButton

This call should be: `buttonGroup.AddButton(component.NewButton("button", action.Payload{}))`